Tools/autotest/apmrover2.py

```diff
@@ -353,7 +353,7 @@ def drive_brake_get_stopping_distance(self, speed):
         self.set_parameter('ATC_ACCEL_MAX', 15)
         self.change_mode("STEERING")
         self.set_rc(3, 2000)
-        self.wait_groundspeed(15, 100)
+        self.wait_groundspeed(15, 100, maintain_target_time=1)
```

Comment:
I still don't understand why we now require the speed to be maintained for 1 second.  That's a behavioural change.  Similarly for the other entry points.

Reply:
that way we unsure that the speed controler do it job correctly and not jitter arround the target.
I am pretty in favor on enforcing the correct speed and time value everywhere ...

Reply:
I could remove that from this pr but it will come after